presto-verifier/src/main/java/com/facebook/presto/verifier/framework/VerifierModule.java

```diff
@@ -167,6 +167,13 @@ public void setInjector(Injector injector)
         }
     }
 
+    @Provides
+    @Singleton
+    public static ScheduledExecutorService createScheduledExecutorService()
+    {
+        return newSingleThreadScheduledExecutor(daemonThreadsNamed("verifier-scheduled-executor-service"));
```

Comment:
is this created for future use or it replaces any existing one?

Reply:
I need one in one of our proprietary modules. But I just thought it might be useful in general to have a shared scheduler for small background tasks in Verifier